core/frontend/src/components/helper.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-simple-table class="helper-table">
+    <thead>
```

Comment:
Forgot to add, but it would be better if this and eventually other helper components stay under `@/components/helper/`. This way we keep the service-specific components separated.